source/qml/Samples/QmlVisualizer/samplecardmodel.cpp

```diff
@@ -114,9 +114,8 @@ QString SampleCardModel::generateQml(const QString& cardQml)
 	{
 		if (genQml.GetElement() == "Frame" && genQml.HasProperty("readonly property bool hasBackgroundImage"))
 		{
-			std::string str = genQml.GetProperty("background");
-
-			genQml.Property("background", std::regex_replace(str, std::regex("source:.*\n"), "source:\"" + getImagePath("Frame") + "\"\n"));
+            std::string str = genQml.GetProperty("background");
+            genQml.Property("background", std::regex_replace(str, std::regex("source.*\n"), "source:\"" + getImagePath("Frame") + "\"\n"));
```

Comment:
if other properties like sourceClipRect or sourceSize is used it will replace them in this case

Reply:
are we using those properties today?

Reply:
no

Reply:
The should be fine as... this code is in visulizer... consumer can opts to read the property as they wish

Reply:
Oh it got what you meant.. changing 👍

Reply:
the same happened with me when changing the * with \* for showcard